pdfminer/pdfdocument.py

```diff
@@ -4,11 +4,20 @@
 import struct
 
 try:
-    from Crypto.Cipher import ARC4, AES
-    from Crypto.Hash import SHA256
+    from cryptography.hazmat.primitives.ciphers import (Cipher, algorithms,
+                                                        modes)
+    from cryptography.hazmat.backends import default_backend
+    AES = True
 except ImportError:
-    AES = SHA256 = None
-    from . import arcfour as ARC4
+    AES = None
```

Comment:
This try-except is not needed anymore. I think installing cryptography is no problem for everyone.